
# Kinova Param
kinova_robotType: 'j2s7s300'

exec_done: 'false'
